packages/clay-pagination/src/PaginationWithBasicItems.tsx

```diff
@@ -105,14 +118,18 @@ const ClayPaginationWithBasicItems = React.forwardRef<HTMLUListElement, IProps>(
 			activePage,
 			alignmentPosition,
 			ariaLabels = {
-				next: 'Next',
-				previous: 'Previous',
+				link: 'Go to page, %PAGE%',
+				next: 'Go to the next page, %NEXT%',
+				previous: 'Go to the previous page, %PREVIOUS%',
```

Comment:
There is already a pattern we have for this, instead of the `%PAGE%` syntax, you can use `Go to page, {0}` and use the `sub` utility from the `@clayui/shared` package. It also works for multiple occurrences.

```js
sub(ariaLabels.link, [
  page,
])
```

Reply:
It looks like the sub method just replaces any digit wrapped in curly brackets. I might be missing something but is there a way we can specify what to sub? I'm wanting to use it for this label "Show pages %START% through %END%".